lib/cereal/grpc/grpc.go

```diff
@@ -41,6 +37,15 @@ func NewGrpcBackendFromConn(domain string, conn *grpc.ClientConn) *GrpcBackend {
 		client: grpccereal.NewCerealClient(conn),
 	}
 }
+
+func (g *GrpcBackend) DefaultTaskPollInterval() time.Duration {
+	return 10 * time.Second
+}
+
+func (g *GrpcBackend) DefaultWorkflowPollInterval() time.Duration {
+	return 2 * time.Second
+}
```

Comment:
I added this in response to a test failure that makes sure we are starting workflows within some bound of their expected start time.

We could alternatively simply set different polling intervals in our tests, but this seemed like an easy way to allow backends to bring their own default values if they need something different.